### PR TITLE
Uifix evidencepage

### LINKED ISSUE #1143: Evidence page consistent type size and style
## Summary

Lead string is bigger than bullets in terms of typeface. 
Make consistent across the board.

Also recommend reducing the size of the '0 files attached' to draw emphasis to more important page elements.

## Design Detail

Screen Shot 2020-01-16 at 12 18 00 AM

### PR DESCRIPTION
**Pull request by amazingphilippe**
# Description

Adjusted type sizes on evidence page. Removed Lead as it did not make sense for it to be there. 

Fixes #1143

- [x] Bug fix

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made any needed changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: f2/src/forms/EvidenceInfoForm.js [hunk 1]
```diff
@@ -108,9 +109,9 @@ export const EvidenceInfoForm = props => {
             <Container>
               {files.map((f, index) => (
                 <React.Fragment key={index}>
-                  <InfoCard mb={4}>
+                  <InfoCard as="article" mb={4}>
                     <Stack spacing={4}>
-                      <H3 fontSize="2xl">{f.name}</H3>
+                      <H1 fontSize="2xl">{f.name}</H1>
```
Comment (sastels) [inline f2/src/forms/EvidenceInfoForm.js]: We can't have another H1 on this page for a11y reasons - can we style the H3 (or an H2) to be the size you want?

Comment (amazingphilippe) [reply f2/src/forms/EvidenceInfoForm.js]: Good catch, I was under the impression that articles needed an h1 as they are self contained.

**Comment by amazingphilippe** [comment]
Can we approve those changes?

**Comment by sastels** [comment]
opps, I didn't notice your fix - let's hold off until we're done testing tomorrow.

**Pull request merged**